serializer/serializer.go

```diff
@@ -30,19 +33,37 @@ func NewSerializer(poller poller.Fetcher) *Serializer {
 // GetSerializedData serializes split and segment data into strings
 func (serializer *Serializer) GetSerializedData() (string, error) {
 	latestData := serializer.poller.GetSplitData()
+	if reflect.DeepEqual(latestData, poller.SplitData{}) {
+		return fmt.Sprintf(emptyCacheLoggingScript), nil
```

Comment:
I think `fmt.Spintf` maybe can be removed here?

Reply:
ah yep, ty!